core/interception.js

```diff
@@ -145,340 +172,446 @@ wsHook.after = function(messageEvent, url)
+
     var safetyDelayPanel = document.getElementById("incognito-safety-delay-option-panel");
     var safetyDelayPanelExpectedHeight = 44; // be careful with this
-    if (readConfirmationsHookEnabled) 
+    if (readConfirmationsHookEnabled)
     {
         getCSSRule('html[dir] .' + UIClassNames.UNREAD_COUNTER_CLASS).style.backgroundColor = 'rgba(9, 210, 97, 0.3)';
         if (safetyDelayPanel != null)
-            Velocity(safetyDelayPanel, { height: safetyDelayPanelExpectedHeight, opacity: 1, marginTop: 15} , 
-                                        { defaultDuration: 200, easing: [.1, .82, .25, 1] });
+            Velocity(safetyDelayPanel, { height: safetyDelayPanelExpectedHeight, opacity: 1, marginTop: 15 },
+                { defaultDuration: 200, easing: [.1, .82, .25, 1] });
     }
-    else 
+    else
     {
         getCSSRule('html[dir] .' + UIClassNames.UNREAD_COUNTER_CLASS).style.backgroundColor = 'rgba(9, 210, 97, 1)';
         if (safetyDelayPanel != null)
-            Velocity(safetyDelayPanel, { height: 0, opacity: 0, marginTop: -10} , { defaultDuration: 200, easing: [.1, .82, .25, 1] });
-        var warningMessage = document.getElementsByClassName("incognito-message").length > 0 ? 
-                            document.getElementsByClassName("incognito-message")[0] : null;
+            Velocity(safetyDelayPanel, { height: 0, opacity: 0, marginTop: -10 }, { defaultDuration: 200, easing: [.1, .82, .25, 1] });
+        var warningMessage = document.getElementsByClassName("incognito-message").length > 0 ?
+            document.getElementsByClassName("incognito-message")[0] : null;
         if (warningMessage != null)
         {
-            Velocity(warningMessage, { scaleY: [0,1], opacity: [0, 1]} , 
-                    { defaultDuration: 300, easing: [.1, .82, .25, 1] });
-            setTimeout(function() {warningMessage.parentNode.removeChild(warningMessage);}, 300);
+            Velocity(warningMessage, { scaleY: [0, 1], opacity: [0, 1] },
+                { defaultDuration: 300, easing: [.1, .82, .25, 1] });
+            setTimeout(function () { warningMessage.parentNode.removeChild(warningMessage); }, 300);
         }
     }
-    
+
     if ('readConfirmationsHook' in options)
     {
         var unreadCounters = document.getElementsByClassName(UIClassNames.UNREAD_COUNTER_CLASS);
-        for (var i=0;i<unreadCounters.length;i++)
+        for (var i = 0; i < unreadCounters.length; i++)
         {
-            unreadCounters[i].className =  UIClassNames.UNREAD_COUNTER_CLASS;
+            unreadCounters[i].className = UIClassNames.UNREAD_COUNTER_CLASS;
         }
     }
 });
 
-document.addEventListener('onReadConfirmationBlocked', function(e) 
+document.addEventListener('onReadConfirmationBlocked', function (e)
 {
     var blockedJid = e.detail;
-    
+
     var chat = getCurrentChat();
-    
+    console.log("onReadConfirmationBlocked")
```

Comment:
This is probably a debug log that can be removed

Reply:
removed 😄